include/libpmemobj++/experimental/radix_tree.hpp

```diff
@@ -616,6 +616,41 @@ radix_tree<Key, Value, BytesView>::radix_tree(InputIt first, InputIt last)
 		emplace(*it);
 }
 
+/* Copy constructor, description todo */
+template <typename Key, typename Value, typename BytesView>
+radix_tree<Key, Value, BytesView>::radix_tree(const radix_tree &m)
+{
+	check_pmem();
```

Comment:
FYI, there's already a test you can use to test this check: https://github.com/pmem/libpmemobj-cpp/blob/39895391980be1b18a946556cafb98dbf7276e4b/tests/map/map_ctor_exception_nopmem.cpp You just need to chnage the ifdef.